src/client/components/general/flex/MenuFlex.tsx

```diff
@@ -0,0 +1,13 @@
+import styled from 'styled-components';
+
+/**
+ * A wrapper around the elements of options above the Courses table , used for
+ * adding secondary layers of flex positioning.
+ */
```

Comment:
I think `MenuFlex` is fine as a name here, but I do think we'll probably also use this for the Schedule page so I don't think we need to specify that it's for the Courses Page in the comment.

Reply:
Ok that makes sense. I updated the comment in [042ce39](https://github.com/seas-computing/course-planner/pull/460/commits/042ce397ad4ba352ec39f907bd00e3573e10c796).